webhook_receiver/settings/__init__.py

```diff
@@ -145,8 +145,16 @@
 CELERY_TASK_ALWAYS_EAGER = not bool(CELERY_BROKER_URL)
 
 DATABASES = {
-    'default': env.db('DJANGO_DATABASE_URL',
```

Comment:
Doesn't this mean that the `DJANGO_DATABASE_URL` envar is now ignored, and the user now has to specify a whole boatload of `DJANGO_DB_*` settings instead? (The latter is exactly what using `DJANGO_DATABASE_URL` and `env.db` avoids.)

```
DB_URL="mysql://foo:bar@localhost:3306/blatch" python3
>>> from environ import Env
>>> env = Env()
>>> env.db('DB_URL')
{'NAME': 'blatch', 'USER': 'foo', 'PASSWORD': 'bar', 'HOST': 'localhost', 'PORT': 3306, 'ENGINE': 'django.db.backends.mysql'}
```

`env.db()` does already give you the dictionary that you are looking for, so what's the reason for throwing that away here?

Reply:
See [here](https://github.com/hastexo/webhook-receiver/pull/10#discussion_r793357542)